contents/data-display/banners/announcement/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Link, CloseButton, HStack, Text } from "@yamada-ui/react"
+import type { FC } from "react"
+
+const Announcement: FC = () => {
```

Comment:
@setodeve 

Is it okay not to memoize with `React.memo()` like other components?

Reply:
@koralle 
I have forgotten about it. I will deal with it on other Issue.